maestro-cli/src/main/java/maestro/cli/command/TestCommand.kt

```diff
@@ -158,6 +158,19 @@ class TestCommand : Callable<Int> {
     )
     private var headless: Boolean = false
 
+    @Option(
+        names = ["--analyze"],
+        description = ["[Beta] Enhance the test output analysis with AI Insights"],
+    )
+    private var analyze: Boolean = false
+
+    @Option(names = ["--apiUrl"], description = ["[Beta] API base URL"])
```

Comment:
other params use `dashes-between-words` instead of `camelCase`

Reply:
Oh i'll adjust to kebab-case since every command in this file is following this pattern, but I inspired these options from the `UploadCommand` do I change them there as well or do i open an issue later for consistency across all commands?